src/main/java/frc/robot/Controllers.java

```diff
@@ -7,7 +7,11 @@ public final class Controllers {
   public static int DRIVER_SEC_PORT = 0;
   public static int DRIVER_TURN = 4;
   public static int DRIVER_SPEED = 1;
-  public static boolean DRIVER_W_BUTTONS = false; // Work in progress, Don't touch please
+  public static int HIGHSPEED = 2;
+  public static int LOWSPEED = 1;// 0.5
+  public static double LOWEREDSPEED = 0.55;
+  public static boolean DRIVER_W_BUTTONS = false;
+  public static int CURRENT_SPEEDLIMIT = 2;
```

Comment:
What's the expected max range for speed limit?  -1 to 1? Or is this a integer that changes between whatever HIGHSPEED and LOWSPEED is?  If the latter, maybe change this to `CURRENT_SPEEDLIMIT = HIGHSPEED` to help clarify the connection.

Reply:
its the numbers of the gears

Reply:
kk, for thing like that, enums tend to be used to generate a list of options and then in code refer to the options by name instead of number like https://www.w3schools.com/java/java_enums.asp